controller/proxy-injector/fake/data/config-proxy.yaml

```diff
@@ -1,66 +1,49 @@
 env:
```

Comment:
Sigh, this file and `config-proxy-init.yaml` are completely unused, so you could remove both of them, and their corresponding consts in `factory.go`:

```diff
diff --git a/controller/proxy-injector/fake/factory.go b/controller/proxy-injector/fake/factory.go
index 10511799..76551299 100644
--- a/controller/proxy-injector/fake/factory.go
+++ b/controller/proxy-injector/fake/factory.go
@@ -14,8 +14,6 @@ import (
 const (
 	DefaultControllerNamespace = "linkerd"
 	DefaultNamespace           = "default"
-	FileProxySpec              = "fake/data/config-proxy.yaml"
-	FileProxyInitSpec          = "fake/data/config-proxy-init.yaml"
 	DefaultNoInitContainer     = false
 )
 
```